### DISCUSSION TITLE
Cursor/column line highlighting gone after latest update

### DISCUSSION THREAD

**Discussion by tommyalatalo** [LazyVim/LazyVim#5198] [Q&A]
Just updated to new lazyvim yesterday, not sure if I jumped over several versions, but either way my cursor and column line highlights are gone.

I have this in my options.lua, so something else is overriding it I guess. Anyone know what's up with this?
```
opt.cursorline = true
```

**Comment by folke** [comment]
no idea. cursorline is enabled in lazyvim by default..

**Comment by dpetka2001** [comment]
Are you using default tokyonight colorscheme or something else? If you use other colorscheme, maybe it doesn't have correct colors for `cursorline` and `cursorcolumn`?

**Reply by tommyalatalo** [reply]
I've been using `sainnhe/gruvbox-material` for quite some time, and haven't changed settings for that as far as I know. I think there was an update for the colorscheme as well, maybe the highlighting has been borked... I don't suppose anyone here is also using it?

**Reply by dpetka2001** [reply]
I use Tokyonight and `cursorline` shows by default because it's already enabled. And if I also enable `cursorcolumn` (which is not enabled by default), I can also see that. 

Maybe better ask at the colorscheme Github repo for some help? I don't use it myself and can't be of further help. You could also just try to set the highlight group `CursorColumn` to a temporary color like this `:hi Cursorcolumn guibg=red` and see if that takes effect. If you can see it, then it's related to the colors of your colorscheme.

**Reply by OneOfOne** [reply]
There's something weird going on with colorschemes in the last few releases.

**Reply by tommyalatalo** [reply]
> I use Tokyonight and `cursorline` shows by default because it's already enabled. And if I also enable `cursorcolumn` (which is not enabled by default), I can also see that.
> 
> Maybe better ask at the colorscheme Github repo for some help? I don't use it myself and can't be of further help. You could also just try to set the highlight group `CursorColumn` to a temporary color like this `:hi Cursorcolumn guibg=red` and see if that takes effect. If you can see it, then it's related to the colors of your colorscheme.

I tried setting the cursorcolumn to red with your suggested command, it doesn't work, still doesn't show up.

**Reply by dpetka2001** [reply]
What does `:verbose set cursorcolumn?` say?

**Reply by tommyalatalo** [reply]
> `:verbose set cursorcolumn`

it doesn't seem to respond with a message, but the cursorcolumn appears after setting it like that.

**Reply by dpetka2001** [reply]
That's because it's not set by default. You have to put in your `options.lua` file `vim.opt.cursorcolumn = true`.
